src/filter-image/extract/extractFilters.ts

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { tags } from '../../xml';
+import { FilterElement, Filters } from '../types';
+import { parse } from './extractFiltersString';
+
+export const extractFiltersCss = (rawFilters?: Filters | string): Filters => {
+  if (!rawFilters) {
+    return [];
+  }
+  if (Array.isArray(rawFilters)) {
+    return rawFilters as Filters;
+  }
+  return parse(rawFilters);
+};
+
+export const mapFilterToComponent = (
+  { name, ...props }: FilterElement,
+  index: number
+) => {
+  return tags[name]
```

Comment:
We update the `tags` with the filters that are implemented yeah?

Reply:
Yes, `tags` are imported from `xml.ts` file, there we have a list of all implemented `SVG` components
```tsx
export const tags: { [tag: string]: ComponentType } = {
  svg: Svg,
  circle: Circle,
  ...,
  filter: Filter,
  feColorMatrix: FeColorMatrix,
};
```